web/cobrands/northamptonshire/assets.js

```diff
@@ -483,4 +483,30 @@ fixmystreet.assets.add($.extend(true, {}, northants_road_defaults, {
     ]
 }));
 
+// Hide form when emergency category used
+function check_emergency() {
+    var relevant_body = OpenLayers.Util.indexOf(fixmystreet.bodies, northants_defaults.body) > -1;
+    var relevant_cat = $('label[for=form_emergency]').length;
+    var relevant = relevant_body && relevant_cat;
+    var currently_shown = !!$('#northants-emergency-message').length;
+    var body = $('#form_category').data('body');
```

Comment:
This does mean if someone who has a council login for another council uses this they'll still be able to report. I'm not sure that's much of an issue in practice though.